course/lesson_04_unit_testing/README.md

```diff
@@ -5,10 +5,11 @@
 
 ### practice projects
 
-1. Write unittest.TestCase tests (2-3) for (The Collatz Sequence) function. 
-Define test data for tests in setUp method. 
+1. Modify (The Collatz Sequence) so that its main function takes an argument and return list with all numbers.
+Write `unittest.TestCase` tests (2-3) for it.
```

Comment:
This is supposed to mean 2 or 3? Give a specific test to implement, so that it's clear how it should be done. For example:
* test that it crashes on input 'not a number'
* test that for input 4 it retuns 2,